README.md

```diff
@@ -274,55 +274,11 @@ Auth0
     }
 ```
 
-### Management API (Users)
```

Comment:
This was here because the MGMT API implemented on these mobile sdks is only meant to do small "User's" stuff, such as get full profile or link accounts. If you remove it from here you might want to add this clarification somewhere below.